pkg/ocm/provider/authorizer/json/json.go

```diff
@@ -104,7 +104,11 @@ func normalizeDomain(d string) (string, error) {
 		return "", err
 	}
 
-	return u.Host, nil
+	normalizedDomain := u.Hostname()
+	if port := u.Port(); port != "" {
+		normalizedDomain += ":" + port
+	}
+	return normalizedDomain, nil
```

Comment:
@fschade is that not part of any other PR? Related to the current breaking ocm setup in web?

Reply:
ok, dropped this  and combinde the commits to glue the changelog to the change